internal/client/service.go

```diff
@@ -698,7 +700,11 @@ func (s *ServiceImpl) UpdateDeploymentFeatureCacheSettings(ctx context.Context,
 	return Patch[DeploymentFeatureCacheSettings](s.client, ctx, "/deployments/"+id+"/featureCache/", req)
 }
 
-func (s *ServiceImpl) UpdateDeploymentRetrainingSettings(ctx context.Context, deploymentID string, req *UpdateRetrainingSettingsRequest) (*RetrainingSettings, error) {
+func (s *ServiceImpl) GetDeploymentRetrainingSettings(ctx context.Context, deploymentID string) (*RetrainingSettingsRetrieve, error) {
+	return Get[RetrainingSettingsRetrieve](s.client, ctx, "/deployments/"+deploymentID+"/retrainingSettings/")
```

Comment:
```suggestion
	return Get[RetrainingSettingsRetrieve](s.client, ctx, fmt.Sprintf("/deployments/%s/retrainingSettings/", deploymentID))
```

Reply:
I would think that `staticcheck` (configured in .golangci.yaml) should throw a warning for all these places. Wondering why it's not 🤔

Reply:
Looks like the work for the follow PR